org.metaborg.spoofax.core/src/main/java/org/metaborg/spoofax/core/analysis/constraint/AbstractConstraintAnalyzer.java

```diff
@@ -275,4 +349,92 @@ private IMessage message(ITerm originatingTerm, IMessageInfo messageInfo, Messag
         };
     }
 
+    protected void flowspecDemoOutput(C context, final IControlFlowGraph<CFGNode> controlFlowGraph) {
```

Comment:
Maybe it is better to add a primitive similar to https://github.com/metaborg/nabl/blob/develop/nabl2/meta.nabl2.java/src/main/java/org/metaborg/meta/nabl2/spoofax/primitives/SG_debug_scope_graph.java, instead of hardcoding this here?

Reply:
Yes, that would be better, I'm looking into that